### ISSUE TITLE
Cannot set class on TextInput input DOM element

### ISSUE THREAD

**Issue by drewbitt** [svelteuidev/svelteui#296]
### What package has an issue

@svelteuidev/core

### A clear and concise description of what the bug is

A `TextInput` when rendered creates something like

```
<div>
  <div>
     <input type="text">
  </div>
</div>
```

When setting `class` on the TextInput, like you would do in Tailwind, you might expect it to apply the classes to the input element (or at least have the option to do so with a similar class prop). However, it sets the classes on the first div element two layers up.

###  Workaround (For now)
You could use some of the classes svelteui is putting on the input, either in an external CSS selector, via css parent-child chaining, or in a :global style tag e.g.
```css
<style>
       // can also scope this with ref
	:global(.svelteui-Input-input) {
		background-color: transparent !important;
	}
</style>
```
However I haven't tested if this classname will always be there in production

### Steps To Reproduce

Use svelteui 0.9

### Do you know how to fix the issue

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

**Comment by drewbitt** [comment]
Note that if you fall back to <Input>, which is of course documented with advise not to use directly, the last class name will be the variant, overwriting all other class set values as it appears last in the list. This forces the use of the headless variant.

**Comment by BeeMargarida** [comment]
Hi, sorry, this got forgotten.
I still have to check this out and see what can be the problem, but thank you so much for this bug report, really detailed.

**Comment by drewbitt** [comment]
I came across a similar scenario today with Modals; the styling of the Modal header, which the `title` text prop is in, cannot be set. I imagine other scenarios exist where inner elements have desired styling. Therefore, I am happy with setting styles on the classes themselves which I outlined ways of doing in my original post, e.g. `.svelteui-Modal-header` or `.svelteui-Input-input`. Rather than adding more props on all elements where they have inner elements - which of course is still a possible solution. However the avenue of direct class styling should be added to the documentation, at least a FAQ outlining where to find css classnames. Svelte seemed to keep the classnames when I ran the production buid with `vite preview`
